deb-arm/Dockerfile

```
@@ -55,6 +55,15 @@ RUN apt-get update && apt-get install -y fakeroot curl git procps bzip2 \
     gettext libtool autopoint autoconf libtool-bin \
     selinux-basics default-jre flex wget
 
+# Ubuntu 16.04 comes with gcc 5.4 by default, which doesn't work with the race detector starting with Go 1.22
+# NOTE: we *could* uninstall gcc 4.8, but the "rvm requirements" run later on would reinstall
+# it and there's not way to prevent that. So leave it be and use update-alternatives to select 9
+RUN apt install -y software-properties-common
+RUN add-apt-repository -y ppa:ubuntu-toolchain-r/test
+RUN apt-get update \
+    && apt-get install -y gcc-9 g++-9 \
+    && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 0
```

nit - you could have only one `RUN` to optimize a bit 
```suggestion
RUN apt install -y software-properties-common \
    && add-apt-repository -y ppa:ubuntu-toolchain-r/test \
    && apt-get update \
    && apt-get install -y gcc-9 g++-9 \
    && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 0
```

---

I copy pasted from the `deb-x64` image but I agree !